index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="styles.css">
+
+  <title>Chitter Challenge</title>
+</head>
+
+<body>
+  <div id="login-container">
+    <input type="text" id="input-username" placeholder="Username"/>
+    <input type="password" id="input-password" placeholder="Password"/>
+    <button id="submit-login">Login</button>
+    <div id="logged-in"></div>
+  </div>
+  <h1>Chitter Challenge</h1>
+  <input type="text" id="input-new-post"/>
+  <button id="add-new-post">Peep!</button>
+  <p>Welcome to Chitter</p>
+  <div id="post-container">
+
+  </div>
+
+  <script type="text/javascript" src="bundle.js"></script>
```

Comment:
Alex mentioned to us that it's good practice to include the script in the head with "Defer" in the tag to do the same thing

Reply:
`<script type="text/javascript" src="bundle.js" defer ></script>'